tests/templates/test_broadcast.py

```diff
@@ -318,23 +321,6 @@ def circuit2():
         built_in = circuit2()
         assert np.allclose(custom, built_in)
 
-    @pytest.mark.parametrize("custom_pattern", [1,
-                                                [1, 2],
-                                                [['a'], ['b']]
-                                                ])
-    def test_exception_custom_pattern_not_valid(self, custom_pattern):
-        """Tests that an exception is raised if the pattern is not a list of lists of integers."""
-
-        dev = qml.device('default.qubit', wires=2)
-
-        @qml.qnode(dev)
-        def circuit():
-            broadcast(unitary=qml.Hadamard, wires=[0, 1], pattern=custom_pattern)
-            return qml.expval(qml.PauliZ(0))
-
-        with pytest.raises(ValueError, match="a custom pattern must be a list"):
-            circuit()
-
```

Comment:
Test not needed any more because wire management checks that the pattern has the correct structure...